docs/src/main/asciidoc/parameter-store.adoc

```diff
@@ -121,6 +121,43 @@ However, starting at `spring-cloud-aws` `2.3`, allows import default aws paramet
 |With `optional` application will start even if there is no parameter found for specified parameter.
 |===
 
+Spring Boot 2.4 changed the ways files are loaded which means profile loading has changed for Spring-Cloud-AWS as well.
+Each document is separated with `#---` and describes one file. Only document in which active profile is equal to `spring.config.activate.on-profile` is used.
```

Comment:
I think we can add a reference to spring boot docs here instead of trying to explain ourselves. since this is already documented.

Reply:
I have put a link to reference docs and to a blog just in case.